weatherapplication.client/src/index.html

```diff
@@ -1,177 +1,297 @@
+            loaderScreenFontSize +
+            "px !important; margin: 0 auto; margin-top: " +
+            loaderScreenMargin +
+            "px; text-align: center'> " +
+            loaderScreenMessage +
+            " </small>";
+
+          // Style
+          data.style =
+            "align-items: center; background-color: rgba(255, 255, 255, .98); display: flex; flex-direction: column; height: " +
+            innerHeight +
+            "px; justify-content: center; left: 0; margin: auto; max-height: 100% !important; max-width: 100% !important; min-height: 100vh; min-width: 100vh; position: fixed; top: 0; transition: " +
+            loaderScreenTransition +
+            "s ease-in-out; width: " +
+            innerWidth +
+            "px; z-index: 2147483647";
 
-    /* Logic
+          // Function
+          // Alpha
+          function alpha() {
+            // Clear Interval
+            clearInterval(metadata);
+          }
+
+          // Test
+          function test() {
+            // Style > Data
+            // Background Color
+            data.style.backgroundColor = "transparent";
+
+            // Opacity
+            data.style.opacity = 0;
+
+            // Set Timeout
+            setTimeout(
+              function () {
+                // Deletion
+                data.remove();
+
+                // Modification > <body> > Style
+                document.body.style = document.body
+                  .getAttribute("style")
+                  .replace("overflow: hidden !important;", "")
+                  .replace("pointer-events: none !important;", "")
+                  .replace("user-drag: none !important;", "")
+                  .replace("user-select: none !important;", "");
+                (document.body.getAttribute("style") || "").trim() ||
+                  document.body.removeAttribute("style");
+              },
+              (+getComputedStyle(data)
+                .getPropertyValue("animation-delay")
+                .replace(/[a-zA-Z]/g, "")
+                .trim() +
+                +getComputedStyle(data)
+                  .getPropertyValue("animation-duration")
+                  .replace(/[a-zA-Z]/g, "")
+                  .trim() +
+                +getComputedStyle(data)
+                  .getPropertyValue("transition-delay")
+                  .replace(/[a-zA-Z]/g, "")
+                  .trim() +
+                +getComputedStyle(data)
+                  .getPropertyValue("transition-duration")
+                  .replace(/[a-zA-Z]/g, "")
+                  .trim()) *
+                1e3 +
+                1000
+            );
+          }
+        }, timeout());
+      }
+
+      /* Logic
             [if:else if:else statement]
     */
-    if (condition())
-      // Update
-      update();
-
-    else {
-      // Initialization > Data
-      var data = setInterval(function () {
-        /* Logic
+      if (condition())
+        // Update
```

Comment:
Useless comments are worse than their absence. I see a lot of them in this file.